test/functional/apps/dashboard_elements/controls/options_list/options_list_allow_expensive_queries_off.ts

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { OPTIONS_LIST_CONTROL } from '@kbn/controls-plugin/common';
+import expect from '@kbn/expect';
+
+import { OPTIONS_LIST_ANIMAL_SOUND_SUGGESTIONS } from '../../../../page_objects/dashboard_page_controls';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const browser = getService('browser');
+
+  const { dashboardControls, timePicker, console, common, dashboard, header } = getPageObjects([
+    'dashboardControls',
+    'timePicker',
+    'dashboard',
+    'console',
+    'common',
+    'header',
+  ]);
+
+  const setAllowExpensiveQueries = async (value: boolean) => {
+    await common.navigateToApp('console');
+    await console.closeHelpIfExists();
+    await console.clearTextArea();
+    await console.enterRequest(
+      '\nPUT _cluster/settings\n{"transient": {"search.allow_expensive_queries": ' + value + '}}'
+    );
+    await console.clickPlay();
+    await header.waitUntilLoadingHasFinished();
+    await browser.refresh();
+  };
+
+  describe('Allow expensive queries setting is off', () => {
+    let controlId: string;
+
+    before(async () => {
+      await setAllowExpensiveQueries(false);
+
+      await common.navigateToApp('dashboard');
+      await dashboard.clickNewDashboard();
+      await dashboard.ensureDashboardIsInEditMode();
+      await timePicker.setDefaultDataRange();
+      await header.waitUntilLoadingHasFinished();
```

Comment:
I was originally trying to navigate to the `Test Options List Control` dashboard that we use for all the other options list tests - however, in doing so, I found out that the search bar on the dashboard listing page uses the `prefix` query, which means that it won't work when `search.allow_expensive_queries` is `false`. 

To get around this, I tried to navigate to the dashboard directly via URL - however, this was causing flakiness for some reason where certain elements were considered detached despite trying things like manually switching to edit mode, switching to edit mode as part of the URL, refreshing the browser on the dashboard, etc. Rather than fighting with this, since the tests were so small, I figured starting fresh was a decent enough solution for the flakiness :shrug: